### PR TITLE
Smallpeice 2021 rules

### PR DESCRIPTION
**Pull request by prophile**
**TODO**:

- [x] match duration
- [x] Tokens in diagram

### CODE REVIEW

File: regulations.tex [hunk 1]
```diff
@@ -3,58 +3,9 @@ \section{Regulations}
...
-          compliance inspection.
-    \item \staff may reinspect your robot and invalidate previous inspections at
-          any time.
-  \end{enumerate}
 \item Any assistance from \staff is provided without guarantees.
 \item Competitors are expected to behave within the spirit of good
       sporting conduct.
-\item Competitors must take reasonable measures to avoid their robot damaging
-      the arena, or anything within it, including other robots. This is a
-      non-contact sport.
```
Comment (WillB97) [inline regulations.tex:25]: We should probably still have some kind of regulation about being a non-contact sport

**Comment by WillB97** [comment]
We discussed in [this issue](https://github.com/sourcebots/competition-simulator/pull/33#discussion_r680393163) the definition of what counts as "in" for a token, we should make sure the rules also agree with this.

File: regulations.tex (hunk 1 above)
Comment (prophile) [reply regulations.tex:25]: c08924f4680cf26f4c37cf91732620dfc3ea918f

**Comment by prophile** [comment]
5ee371e404254d832200eed8adde97a18de78ca3

File: specs.tex [hunk 2]
```diff
@@ -36,56 +25,53 @@ \subsection{Arena}
...
 \refstepcounter{SpecID}
-\label{spec:tokens}
+\label{spec:forklift}
 
 \begin{enumerate}
-  \item Tokens are cuboids with side length \SI{150}{mm} $\pm$ \SI{25}{mm}.
-  \item The exterior surface of a token has an AprilTag marker printed upon it. The marker is identical on all faces.
-  \item All tokens belonging to the same corner will have the same marker ID.
-  \item Tokens are arranged as indicated in Figure~\ref{fig:arena}.
+  \item TODO: Fetch the forklift dimensions from the simulator.
```
Comment (RealOrangeOne) [inline specs.tex:42]: To do

File: specs.tex [hunk 3]
```diff
@@ -36,56 +25,53 @@ \subsection{Arena}
...
+\subsection{Crane}
 \refstepcounter{SpecID}
-\label{spec:marker}
+\label{spec:crane}
 
 \begin{enumerate}
-  \item A `marker' is a square fiducial marker that is a member of the AprilTag 36H11 marker set.
-  \item Every marker has a numeric identifier.
-  \item Marker IDs are assigned according to Table~\ref{tab:markers}.
+  \item TODO: Fetch the crane dimensions from the simulator.
```
Comment (RealOrangeOne) [inline specs.tex:64]: To do

File: specs.tex [hunk 4]
```diff
@@ -36,56 +25,53 @@ \subsection{Arena}
...
-  \item Tokens are cuboids with side length \SI{150}{mm} $\pm$ \SI{25}{mm}.
-  \item The exterior surface of a token has an AprilTag marker printed upon it. The marker is identical on all faces.
-  \item All tokens belonging to the same corner will have the same marker ID.
-  \item Tokens are arranged as indicated in Figure~\ref{fig:arena}.
+  \item TODO: Fetch the forklift dimensions from the simulator.
+  \item The forklift is equipped with the following sensors:
+  \begin{enumerate}
+    \item Magnetic compass.
+    \item Gyroscope.
+    \item Accelerometer.
```
Comment (RealOrangeOne) [inline specs.tex:47]: I don't believe we have an Accelerometer

File: specs.tex [hunk 5]
```diff
@@ -7,26 +7,15 @@ \subsection{Arena}
 \label{spec:arena}
 
 \begin{enumerate}
-  \item The arena floor is an \SI{8.4}{m} $\times$ \SI{8.4}{m} rectangle. The
-        tolerance of these two dimensions is $\pm$ \SI{250}{mm}.
-  \item The floor of the arena is carpeted.
+  \item The arena floor is a \SI{6000}{mm} $\times$ \SI{12000}{mm} rectangle.
```
Comment (RealOrangeOne) [inline specs.tex:10]: Probably also worth noting there are beacons around the arena which can be used for localization.

File: game-rules.tex [hunk 6]
```diff
@@ -3,51 +3,27 @@ \section{Game Rules}
...
-%   \item At the start of each match, teams will be permitted to lean into the
-%         arena and start their robots.
-  \item Each match lasts $120$ seconds.
+        Specification~\ref{spec:arena}. The objective is to collect containers
+        and deposit them in scoring zones.
+  \item The game is played by two teams, each team has two robots: a crane and
+        a forklift.
+  \item The forklift is used to pick up loose containers and move them to the
+        docking area. Each team's forklift starts in their starting area.
+  \item The arena contains 16 containers. The containers are labeled with
```
Comment (RealOrangeOne) [inline game-rules.tex:12]: With https://github.com/sourcebots/competition-simulator/pull/67, 2 tokens now start in the docking area, so there's something for the cranes to do at the start.

This also means there are now 18 tokens.

**Comment by prophile** [comment]
[rules.pdf](https://github.com/sourcebots/smallpeice-rules/files/6992572/rules.pdf)

File: specs.tex (hunk 5 above)
Comment (prophile) [reply specs.tex:10]: First pass done, will add full details in a rules update.

Comment (prophile) [reply specs.tex:10]: Fully resolved by amendment 2.

**Pull request merged**